CONTRIBUTING.md

```diff
@@ -168,6 +170,29 @@ Please remember to:
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels ((enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/aws-step-functions-data-science-sdk-python/labels/help%20wanted) issues is a great place to start.
 
 
+## Setting Up Your Development Environment
+
+Setting up your IDE for debugging your tests locally will save you a lot of time.
```

Comment:
please include a note with the version that these instructions apply to / were tested on.

Reply:
Version of the data science SDK? Like 2.2.0? They should apply to all versions

Reply:
Were you referring to the PyCharm versions @shivlaks? We tested it on 2 different PyCharm versions, I can definitely add the info here.

Reply:
I meant the PyCharm version

Reply:
> I meant the PyCharm version

Ah - makes sense